docs/index.html

```diff
@@ -0,0 +1,135 @@
+	
+	<title>Made in A2 - Companies</title>
+	
+	<script src="https://unpkg.com/vue/dist/vue.js"></script>
+  <!-- <script src="https://unpkg.com/http-vue-loader"></script> -->
+	<script src="https://unpkg.com/vue-router/dist/vue-router.js"></script>
+</head>
+
+<body>
+
+  <div class="top-bar">
+
+    <div class="tabs">
+      <a href="/"          class="tab">Companies</a>
+      <a href="/community" class="tab">Community</a>
+      <a href="/jobs"      class="tab">Jobs</a>
+      <a href="/calendar"  class="tab">Calendar</a>
+    </div>
+    
+    <h1>Made in A2</h1>
+    
+    <a href="/" class="go-home">
+      <img src="/images/a2-whiteboard.png" alt="Made in A2" class="logo" />
+    </a>
+    
+  </div>
+
+  <div class="wrapper" id="app">
+    <!-- app goes here -->
+    <router-view></router-view>
+  </div>
+	
+  <div class="footer">
+    <p>
+      <a href="https://github.com/MadeInA2/madeina2" target="_blank">creators</a>
+      - 
+      <a href="https://twitter.com/MadeInA2" target="_blank">twitter</a>
+      - 
+      <a href="https://github.com/MadeInA2/madeina2" target="_blank">source</a>
+    </p>
+	</div>
+	
+	<!-- vendor -->
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+  <script src="/scripts/vendor/async.min.js"></script>
+  <script src="/scripts/vendor/stickyfill.min.js"></script>
+  <script src="/scripts/vendor/moment.min.js"></script>
+	<script src="/scripts/vendor/fullcalendar.min.js"></script>
+	
+	<!-- main js-->
+	<script src="/scripts/main.js"></script>
+	
+  <!-- a2 data -->
+  <script src="/data/companies.js"></script>
+  <script src="/data/communities.js"></script>
+
+  <!-- pages -->
+  <script src="/pages/companies.vue.js"></script>
+  <script src="/pages/community.vue.js"></script>
+  <script src="/pages/jobs.vue.js"></script>
+  <script src="/pages/calendar.vue.js"></script>
+  <script src="/pages/slack.vue.js"></script>
+
+  <script>
+
+    // page routes
+    const router = new VueRouter({
+      routes: [
+        { path: '/',          component: Companies },
+        { path: '/community', component: Community },
+        { path: '/jobs',      component: Jobs },
+        { path: '/calendar',  component: Calendar },
+        { path: '/slack',     component: Slack }
+      ],
+      mode: 'history',
+    })
+
+    // init vue.js
+    const app = new Vue({
+      router: router,
+    }).$mount('#app')
+
+    // TODO - remove
+    // switch (window.location.pathname.split('/')[1]) {
+    //   case '':
+    //     document.getElementById("companies-tab").className = "tab active";
+    //     break;
+    //   case 'community':
+    //     document.getElementById("community-tab").className = "tab active";
+    //     break;
+    //   case 'jobs':
+    //     document.getElementById("jobs-tab").className = "tab active";
+    //     break;
+    //   case 'calendar':
+    //     document.getElementById("calendar-tab").className = "tab active";
+    //     break;
+    // }
```

Comment:
**nitpick¹:** I wouldn’t commit and push commented code that’s intended for removal. It could distract from intent, so let’s just remove it :-)

---

¹ https://conventionalcomments.org/